lang/funcs/funcgen/config.go

```diff
@@ -24,7 +24,16 @@ import (
 )
 
 type config struct {
-	Functions functions `yaml:"functions"`
+	Packages gopackages `yaml:"packages"`
+}
+
+type gopackages []gopackage
+
+type gopackage struct {
+	Name      string   `yaml:"name"`
```

Comment:
We should docstring each of these fields. See here for an example: https://github.com/purpleidea/mgmt/blob/d5074871c78eacb4616127af074a7ede923d352c/engine/resources.go#L168

Reply:
done